docs/api/versioning.rst

```diff
@@ -0,0 +1,37 @@
+.. _kinto-versioning:
+
+Versioning
+##########
+
+
+Kinto implements an :term:`HTTP API` that can be consumed by clients like
+`Kinto.js <http://kintojs.readthedocs.org/>`_.
+
+The :term:`HTTP API` is described :ref:`here <kinto-api-endpoints>` and is
+subject to changes. It follows the :term:`Cliquet Protocol`.
+
+When the :term:`HTTP API` is changed, its version is incremented.
+The :term:`HTTP API` version follows a :term:`Semantic Versioning`
+pattern and uses this rule to be incremented:
```

Comment:
We should add a note that we'll observe semver only for major and minor, not patch.